app/views/layouts/application.html.erb

```diff
@@ -42,6 +42,13 @@
               <% end %>
             </ul>
           </li>
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><%= current_user.name if current_user %><span class="caret"></span></a>
+            <ul class="dropdown-menu">
+              <li><%= link_to "Logout", "/logout", method: :delete %></li>
+              <li><%= link_to "Dashboard", "/dashboard" %></li>
+            </ul>
+          </li>
```

Comment:
Of course, as I read further through the code, my previous comment is addressed. 😊 This looks good. 👍